src/Middleware/WebSockets/ref/Microsoft.AspNetCore.WebSockets.netcoreapp3.0.cs

```diff
@@ -41,14 +41,8 @@ public static partial class Constants
     {
         public static partial class Headers
         {
-            public const string Connection = "Connection";
             public const string ConnectionUpgrade = "Upgrade";
-            public const string SecWebSocketAccept = "Sec-WebSocket-Accept";
```

Comment:
This seems like a slightly worse breaking change because it straight up removes the constants. However, it is in the internal namespace. Can you try making this class internal too?

Reply:
Internalized